blocks/api/post.pegjs

```diff
@@ -17,10 +17,25 @@ WP_Block_List
   = WP_Block*
 
 WP_Block
-  = WP_Block_Void
+  = WP_Tag_More
+  / WP_Block_Void
   / WP_Block_Balanced
   / WP_Block_Html
 
+WP_Tag_More
+  = "<!--" WS* "more" customText:(WS+ text:$((!(WS* "-->") .)+) { return text })? WS* "-->" noTeaser:(WS* WP_Tag_NoTeaser)?
```

Comment:
actually, I think I can say that I would prefer we enforce the documented syntax. we're already making a break in some fashion with the old, plus when someone loads a post with an incorrect `<!--noteaser-->` tag it will just disappear I think, then they can see the option on the `more` block.

just musing thoughts here